packages/app/src/cli/utilities/developer-platform-client/shopify-developers-client.ts

```diff
@@ -134,30 +163,57 @@ export class ShopifyDevelopersClient implements DeveloperPlatformClient {
   }
 
   async organizations(): Promise<Organization[]> {
-    return [ORG1]
+    const organizationsResult = await businessPlatformRequest<OrganizationsQuerySchema>(
+      OrganizationsQuery,
+      await this.businessPlatformToken(),
+    )
+    return organizationsResult.currentUserAccount.organizations.nodes.map((org) => ({
+      id: Buffer.from(org.id, 'base64').toString('ascii').match(/\d+$/)![0],
+      businessName: org.name,
+    }))
   }
 
   async orgFromId(orgId: string): Promise<Organization | undefined> {
-    if (orgId === '1') return ORG1
+    const gid = `gid://organization/Organization/${orgId}`
+    const base64Id = Buffer.from(gid).toString('base64')
```

Comment:
Here you can see the opposite function: take a numeric ID, convert to a gid and base64-encode.